Source/Admin/Models/TransactionRequestParams+Admin.swift

```diff
@@ -30,6 +30,12 @@ extension TransactionRequestCreateParams {
     ///   - allowAmountOverride: Allow or not the consumer to override the amount specified in the request
     ///                          This needs to be true if the amount is not specified
     ///   - maxConsumptionsPerUser: The maximum number of consumptions allowed per unique user
+    ///   - consumptionIntervalDuration: The duration (in milliseconds) during which the consumptionIntervalDuration and
```

Comment:
Is `which the consumptionIntervalDuration` should be `which the maxConsumptionsPerIntervalPerUser`?